docs/_docs/monitoring-metrics/custom-metrics.adoc

```diff
@@ -0,0 +1,66 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+= Custom Metrics
+
+WARNING: This feature is experimental and may change in future releases.
+
+Ignite provides various internal link:monitoring-metrics/new-metrics.adoc[metrics]. However, these metrics might
+not be enough. User can design and publish own, custom metrics. Custom Metrics base on
+link:monitoring-metrics/new-metrics-system.adoc[Metric System].
+
+[NOTE]
+====
+Custom Metrics are local and are bound only to the local node.
+====
+
+== Custom metric creation.
+
+To register a custom metric, you need to add new link:monitoring-metrics/new-metrics-system#registry[registry] first.
+After that, metrics can be added to this registry.
+
+=== Custom metric registry.
+
+You can create custom metric registries via `IgniteMetrics` interface which is obtained by `Ignite.metrics()`.
+
+`IgniteMetric` interface has only a couple of methods:
+
+* `MetricRegistry getOrCreate(String registryName)` provides new or existing custom metric registry.
+* `void remove(String registryName)` removes entire custom metric registry.
+
+
+=== Custom metric creation.
+
+To register new own metric, use `MetricRegistry` interface which is obtained by `IgniteMetrics.getOrCreate(...)`.
```

Comment:
To register a new custom metric, use the `MetricRegistry` interface which is obtained by `IgniteMetrics.getOrCreate(...)`.